public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/reducer.test.tsx

```diff
@@ -113,7 +113,7 @@ describe('Query and expressions reducer', () => {
       queries: [alertQuery],
     };
 
-    const newState = queriesAndExpressionsReducer(initialState, addNewExpression());
+    const newState = queriesAndExpressionsReducer(initialState, addNewExpression(ExpressionQueryType.math));
```

Comment:
Why we need this change?

Reply:
because before the change, when adding a new expression we were not passing the type, as we always were using ExpressionQueryType.math. Now, we need to pass explicitly the type, as the type is selected before in the button.